### PR TITLE
add new maintainers, photos, bios to the maintainers page

### PR DESCRIPTION
**Pull request by j-loreaux**
--- 

I have only added myself here for now, but I thought other new maintainers would just pile onto this PR with their info.

### CODE REVIEW

File: data/teams.yaml [hunk 1]
```diff
@@ -91,6 +94,7 @@
     - Kexing Ying
     - Aaron Anderson
     - Kevin Buzzard
+    - Eric Rodriguez
```
Comment (eric-wieser) [inline data/teams.yaml:97]: @ericrbg; do you want a middle name here?

Comment (ericrbg) [reply data/teams.yaml:97]: My second surname Boidi would be great :)

Comment (eric-wieser) [reply data/teams.yaml:97]: Feel free to add a suggestion in github, then I'll commit and merge

Comment (ericrbg) [inline data/teams.yaml:97]:
```suggestion
    - Eric Rodriguez Boidi
```

**Pull request merged**